### ISSUE TITLE
Crashing after pressing finish

### ISSUE THREAD

**Issue by shlnz** [guardianproject/haven#147]
As the title says, the app crashes after clicking finish.

```
mako:/ $ getprop ro.product.model
Nexus 4
mako:/ $ uname -a
Linux localhost 3.4.0-lineageos-gc9b5003 #1 SMP PREEMPT Thu Dec 21 13:39:45 CET 2017 armv7l

```

Logcat:
```
--------- beginning of main
12-28 00:47:58.814  5672  5672 I art     : Late-enabling -Xcheck:jni
12-28 00:47:58.951  5672  5672 I MultiDex: VM with version 2.1.0 has multidex support
12-28 00:47:58.951  5672  5672 I MultiDex: Installing application
12-28 00:47:58.951  5672  5672 I MultiDex: VM has multidex support, MultiDex support library is disabled.
12-28 00:47:59.023  5672  5672 W art     : Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
12-28 00:47:59.204  5672  5672 D Main    : onCreate
12-28 00:47:59.520  5672  5672 I Sugar   : domain class : Event
12-28 00:47:59.521  5672  5672 I Sugar   : domain class : EventTrigger
12-28 00:47:59.521  5672  5672 I Sugar   : Create table if not exists
12-28 00:47:59.521  5672  5672 D Sugar   : Fetching properties
12-28 00:47:59.524  5672  5672 I Sugar   : Creating table EVENT
12-28 00:47:59.526  5672  5672 I Sugar   : Create table if not exists
12-28 00:47:59.527  5672  5672 D Sugar   : Fetching properties
12-28 00:47:59.529  5672  5672 I Sugar   : Creating table EVENT_TRIGGER
12-28 00:47:59.567  5672  5672 D SQL Log : SQLiteQuery: SELECT * FROM EVENT ORDER BY id DESC
12-28 00:47:59.830  5672  5704 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 09/02/15, 76f806e, Ibddc658e36
12-28 00:47:59.837  5672  5704 I OpenGLRenderer: Initialized EGL, version 1.4
12-28 00:47:59.838  5672  5704 D OpenGLRenderer: Swap behavior 1
12-28 00:47:59.844  5672  5704 W Adreno-ES20: <get_gpu_clk:229>: open failed: errno 13
12-28 00:51:34.615  5672  5672 I CameraFragment: Sensitivity set to High
12-28 00:51:34.855  5672  5672 D AndroidRuntime: Shutting down VM
--------- beginning of crash
12-28 00:51:34.857  5672  5672 E AndroidRuntime: FATAL EXCEPTION: main
12-28 00:51:34.857  5672  5672 E AndroidRuntime: Process: org.havenapp.main, PID: 5672
12-28 00:51:34.857  5672  5672 E AndroidRuntime: java.lang.RuntimeException: setParameters failed
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.hardware.Camera.native_setParameters(Native Method)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.hardware.Camera.setParameters(Camera.java:2015)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at org.havenapp.main.sensors.motion.Preview.surfaceCreated(Preview.java:214)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.SurfaceView.updateWindow(SurfaceView.java:618)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.SurfaceView$3.onPreDraw(SurfaceView.java:161)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:944)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2205)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1254)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6344)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:874)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:686)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:621)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
12-28 00:51:34.857  5672  5672 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
12-28 00:51:39.858  5672  5672 I Process : Sending signal. PID: 5672 SIG: 9
```

I hope the information is sufficient.

**Comment by fat-tire** [comment]
I think/hope this is fixed by #145 - are you using a recent build?

**Comment by shlnz** [comment]
I am using the fdroid Version, I just updated and it is working fine now. Thank you
